sources/api/schnauzer/tests/v2_parse_success_templates.rs

```diff
@@ -164,9 +163,14 @@ fn succeeds_05_unambiguous_delims() {
             name: "beagle".to_string(),
             version: "+++\n".to_string(),
             helpers: vec![],
-        },
+        }, ExtensionRequirement {
+            name: "helperutil".to_string(),
+            version: "v1".to_string(),
+            helpers: vec!["join_map".to_string()],
+        }
     };
-    let expected_body = "{{ settings.beagle.howl }}\n".to_string();
+    let expected_body =
+        "{{ join_map \"=\" \",\" \"fail-if-missing\" settings.beagle }}\n".to_string();
```

Comment:
Use Rust raw strings to make these more legible (though it get's a little weird with the newline).

```suggestion
       r# "{{ join_map "=" "," "fail-if-missing" settings.beagle }}
"#r.to_string();
```

Reply:
I originally did this with raw strings and converted it to this because of the newline :(